### ISSUE TITLE
Invalid attribute name `arial-label` being output by pagination component

### ISSUE THREAD

**Issue by kruncher** [ONSdigital/design-system#2148]
The main `ons-pagination` block element has the invalid attribute `arial-label` which is a typo, it should be `aria-label`.
